playbook.yml

```diff
@@ -7,3 +7,6 @@
   - include_role:
       name: "fio-bench"
     when: fio is defined and fio.clients > 0
+  - include_role:
+      name: sysbench
+    when: sysbench is defined and sysbench.tests is defined sysbench.tests is not none
```

Comment:
This condition can be then updated to also look at sysbench.enabled

Reply:
Well not really, you need to check if sysbench is defined first otherwise you would get an undefined var error.